src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,154 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.InvalidUserDataException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static StorageDao storageDao;
+    private static RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void clearUp() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void registrationService_loginInStorage_ok() {
```

Comment:
```suggestion
    void register_validData_Ok() {
```
![image](https://github.com/mate-academy/jv-registration-validation/assets/20326858/c69561c5-f56d-48d6-a367-81065b5b1ff2)